### PR TITLE
Add ToC and policies links to footer

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Add the following footer links
- Terms and conditions
- Privacy policy
- Cookie policy

Desktop
Screenshot 2023-04-20 at 16 07 37

Mobile
Screenshot 2023-04-20 at 16 08 40

### CODE REVIEW

File: src/components/Layout/Footer.tsx [hunk 1]
```diff
@@ -1,69 +1,118 @@
-import styled from 'styled-components';
-import Link from 'next/link'
-import { Color, Media } from '@src/const/styles/variables'
-import { scrollToClickedElement } from '@src/lib/analytics/events';
+import styled from "styled-components";
+import Link from "next/link";
+import { Color, Media, Font } from "@src/const/styles/variables";
+import { Section } from "@src/const/styles/pages";
+import { Logo } from "./Header";
```
Comment (shoom3301) [inline src/components/Layout/Footer.tsx:5]: Nitpick: It's pretty weird, that `Footer` depends on `Header`. Maybe make it sense to move `Logo` to another file? Like a `commonComponents.tsx`

File: src/const/menu.ts [hunk 2]
```diff
@@ -8,7 +8,18 @@ export const mainMenu = [
...
-  {id: 1, title: 'Questions & Support', url: social.telegram.url, target: '_blank', onClick: openExternalLinkClickedElement },
+  {
+    label: 'Support', links: [      
+      {id: 0, title: 'Documentation', url: `/${CONFIG.url.docs}` },
+      {id: 1, title: 'Questions & Support', url: social.telegram.url, target: '_blank', onClick: openExternalLinkClickedElement },
+    ]
+  },
+  {
+    label: 'About', links: [      
+      {id: 0, title: 'Terms and Conditions', url: 'https://swap.cow.fi/#/terms-and-conditions', type: 'external' },
```
Comment (shoom3301) [inline src/const/menu.ts:19]: I think we should add `target: _blank` for these links as well

**Comment by elena-zh** [comment]
Hey @nenadV91 , can we open these links in another tab? 
Thanks

**Comment by elena-zh** [comment]
> Hey @nenadV91 , can we open these links in another tab? Thanks

It is fixed in #53 :)

File: src/components/Layout/Footer.tsx (hunk 1 above)
Comment (nenadV91) [reply src/components/Layout/Footer.tsx:5]: Moved to `components/common/Logo`

**Comment by nenadV91** [comment]
@elena-zh will be fixed in the following PR's

**Pull request merged**